pages/people/[id]/warning-notes/[warningNoteId]/[[...stepId]].tsx

```diff
@@ -18,21 +19,17 @@ const ReviewWarningNote = (): React.ReactElement => {
   const confirmation = asPath.includes('confirmation');
   const { user } = useAuth() as { user: User };
   const onFormSubmit = useCallback(
-    (person: Resident) => async ({
-      form_name,
-      ...formData
-    }: Record<string, unknown>) => {
-      console.log({
-        personId: person.id,
-        firstName: person.firstName,
-        lastName: person.lastName,
-        contextFlag: person.contextFlag,
-        dateOfBirth: person.dateOfBirth,
-        workerEmail: user.email,
-        formNameOverall:
-          person.contextFlag === 'A' ? 'ASC_case_note' : 'CFS_case_note',
-        formName: form_name,
-        caseFormData: JSON.stringify(formData),
+    (person: Resident) => async ({ ...formData }: Record<string, unknown>) => {
+      updateWarningNote(warningNoteId, {
+        warningNoteId,
+        reviewedBy: user.email,
+        endedBy: formData.reviewDecision === 'No' ? user.email : undefined,
+        endedDate:
```

Comment:
I think you asked for feedback on this in your PR notes. I might have said `endDate` is a little clearer, but I think it's a small thing. It probably makes most sense to remain consistent with naming we have elsewhere in the codebase and backend, or the terminology used by the people using the tool.

Reply:
I agree, but this feature is like 2 months that has been blocked by the BE and it's the 4th time that we agreed on naming and then they changed them. So we decided to align ourselves to their naming convention, I know it's crazy but this feature has been a real nightmare.